src/metrics.jl

```diff
@@ -136,6 +138,46 @@ cityblock(a::AbstractVector, b::AbstractVector) = evaluate(Cityblock(), a, b)
 evaluate{T <: Number}(dist::Cityblock, a::T, b::T) = abs(a-b)
 cityblock{T <: Number}(a::T, b::T) = evaluate(Cityblock(), a, b)
 
+# Jaccard
+
+function evaluate(dist::Jaccard, a::AbstractVector, b::AbstractVector)
+    n = get_common_len(a, b)::Int
+    numerator = 0
+    denominator = 0
+    for i = 1:n
+        @inbounds ai = a[i]
+        @inbounds bi = b[i]
+        if ai > bi
+          numerator += bi
+          denominator += ai
+        else
+          numerator += ai
+          denominator += bi
+        end
+    end
+    1 - (numerator / denominator)
+end
+
+jaccard(a::AbstractVector, b::AbstractVector) = evaluate(Jaccard(), a, b)
+
+# Tanimoto
+
+function evaluate(dist::RogersTanimoto, a::AbstractVector, b::AbstractVector)
```

Comment:
Since this metric only makes sense for bool arrays it should probably be:

```jl
function evaluate{T <: Bool}(dist::RogersTanimoto, a::AbstractVector{T}, b::AbstractVector{T})
```